phpunit.xml.dist

```diff
@@ -0,0 +1,18 @@
+<phpunit printerClass="ScriptFUSION\PHPUnitImmediateExceptionPrinter\ImmediateExceptionPrinter"
```

Comment:
I'd be mildly cautious of deviating from the SilverLeague standards

Reply:
The usage of a printer is not really something that should be "standardized", and if you want to, I strongly suggest the usage of `ScriptFUSION`, it's way better than the default printer.